symengine/utilities/teuchos/Teuchos_RCPNode.hpp

```diff
@@ -649,7 +652,11 @@ class TEUCHOS_LIB_DLL_EXPORT ActiveRCPNodesSetup {
   /** \brief . */
   ActiveRCPNodesSetup();
   /** \brief . */
+#ifdef TEUCHOS_DEBUG
+  ~ActiveRCPNodesSetup() noexcept(false);
+#else
   ~ActiveRCPNodesSetup();
+#endif
```

Comment:
Can you make this consistent with the others by wrapping only `noexcept(false)`?